devops/staging/values.yaml

```diff
@@ -7,6 +7,7 @@ iam-cache-server-helm:
 
   image:
     repository: "ghcr.io/energywebfoundation/ssi-hub"
+    tag: dce9b11b-2242-4b17-8087-e8bfbcb98d98
```

Comment:
I want to merge `develop` to `release` to get a new stable release tomorrow and then use this release instead of this tag

Reply:
based on what you said, i think you don't need to over-write this tag

Reply:
What do you mean @vickenliu ? I think I do need to specify which image version we want to run in the staging env. But maybe I'm missing my own point 😆